LenovoLegionToolkit.Lib/Utils/Compatibility.cs

```diff
@@ -24,6 +24,7 @@ public static class Compatibility
             "16ACHg6", // Legion 7 - AMD, nVidia
             "16ITHg6", // Legion 7 - Intel, nVidia
             "15IMHg05", // Legion 7 - Intel, nVidia
+            "15ACH6" // Ideapad Gaming 3 - AMD, nVidia
```

Comment:
Missing comma :)

Reply:
> Missing comma :)

thats what happens when you edit files at 3am , thanks for letting me know!